examples/data-sources/ekscluster/cluster.tf

```diff
@@ -0,0 +1,5 @@
+data "tanzu-mission-control_ekscluster" "tf_eks_cluster" {
```

Comment:
Please add the necessary comment stating the resource type and operation, both in data-source and resource. Also in the resource templates.